client/my-sites/customer-home/main.jsx

```diff
@@ -325,24 +325,126 @@ class Home extends Component {
 		);
 	}
 
+	renderSiteTools() {
```

Comment:
Good idea to move this to its own function. Would it make sense to move the `Card`/`FoldableCard` logic here as well? Then the only output in the final render function would be `{ this.renderSiteTools() }` and everything is kept in one place. Not a blocker, IMO.